src/main/java/io/fabric8/elasticsearch/plugin/acl/SearchGuardRolesMapping.java

```diff
@@ -96,7 +96,7 @@ public void syncFrom(UserProjectCache cache, final String userProfilePrefix){
 				builder.addUser(projectRoleName, username);
 			}
 			
-			if ( cache.isClusterAdmin(username) ) {
+			if ( cache.isOperationsUser(username) ) {
```

Comment:
For our needs, is an operations user who could be a cluster-admin or cluster-reader the same 'ADMIN_ROLE'?  Do we have concerns that a cluster-reader may have access to logs they should not?